python/lsst/ts/Dome/mock_controller.py

```diff
@@ -99,12 +99,12 @@ def __init__(
         self.current_tai = 0
 
         # Variables to hold the status of the lower level components.
-        self.amcs = mock_llc_statuses.AmcsStatus()
-        self.apscs = mock_llc_statuses.ApscsStatus()
-        self.lcs = mock_llc_statuses.LcsStatus()
-        self.lwscs = mock_llc_statuses.LwscsStatus()
-        self.moncs = mock_llc_statuses.MoncsStatus()
-        self.thcs = mock_llc_statuses.ThcsStatus()
+        self.amcs = None
```

Comment:
Good suggestions regarding improving the readability of the code. I use these acronyms on purpose since they are used by EIE so it makes it much easier to relate them to their counterparts. I have added the explanations of them in the main doc string of the MockDomeController class. And I agree that adding a _status suffix improves readability as well so I did that.